app/src/components/forms/roles/AddRoleCategoryForm.vue

```diff
@@ -57,7 +71,8 @@
             }
           "
         >
-          <IconMinus />
+          <!--<IconMinus />-->
```

Comment:
```suggestion
```
this one not used? maybe we can delete it

Reply:
Okay, thanks will remove those